### ISSUE TITLE
Multi-line function arguments

### ISSUE THREAD

**Issue by Eathox** [42School/norminette#43]
Norminette is inconstant with how it treats multi-line function arguments, as when using this header file:
```h
#ifndef FT_EXAMPLE_H
# define FT_EXAMPLE_H

void	ft_foobar(
	char really_long_variable_name_for_the_purposes_of_this_example,
	char the_second_one_wouldent_fit_on_the_same_line
);

#endif
```
> ft_example.h: KO!
	TOO_FEW_TAB          (line:   5, col:   1):	Missing tabs for indent level

This can be address by adding more tabs in front of the arguments:

```h
void	ft_foobar(
			char really_long_variable_name_for_the_purposes_of_this_example,
			char the_second_one_wouldent_fit_on_the_same_line
);
```
However then the `);` also has to be moved:
> TOO_FEW_TAB          (line:   7, col:   1):	Missing tabs for indent level

This violates a norm rule so its not a solution:
> NO_SPC_BFR_PAR       (line:   7, col:  13):	Extra space before parenthesis (brace/bracket)

The only working 'configuration' is to move the `);` one line up:
```h
void	ft_foobar(
			char really_long_variable_name_for_the_purposes_of_this_example,
			char the_second_one_wouldent_fit_on_the_same_line);
```
> ft_example.h: OK!

All of this however is not an issue in the function it self:
```c
#include "ft_example.h"

#include <stdio.h>

void	ft_foobar(
	char really_long_variable_name_for_the_purposes_of_this_example,
	char the_second_one_wouldent_fit_on_the_same_line
)
{
	printf("%c%c",
		really_long_variable_name_for_the_purposes_of_this_example,
		the_second_one_wouldent_fit_on_the_same_line
	);
}
```
> ft_example.c: OK!

As far as I can see the first header file 'configuration' should not be violating any norm-rules, is the most readable and is consistent with how its setup for functions in c files.

**Comment by Tishj** [comment]
This should really be fixed, but for the mean time, the "fix" is to put the `,` on the next line

**Comment by gmzorz** [comment]
multi line length checking seems a bit flawed as well. `    nml_inv = vec3_sub((t_vec3){1.0, 1.0, 1.0}, vec3_div_s(
                vec3_mult(cyl->diff, cyl->diff), (void)vec3_dot(cyl->diff, cyl->diff)));` is considered OK but is 87 columns long

**Comment by Eathox** [comment]
@Tishj this is currently not a working 'fix' as it still results in the same error:
> TOO_FEW_TAB (line: 5, col: 1): Missing tabs for indent level 

This 'configuration' should (from what I know) actually also be causing an other norm-error as comma's aren't allowed to be at the start of a line.

@gmzorz The line you provided does return a `LINE_TOO_LONG` norm-error for me. 
I would suggest that you open an other issue with more details on this matter. Or add a comment with more details about this case to the already on going issue #40

**Comment by N0ich** [comment]
Hello,
@gmzorz as Eathox said, it would be better if you opened a ticket for your issue.
Regarding the main issue:
```void	ft_foobar(
			char really_long_variable_name_for_the_purposes_of_this_example,
			char the_second_one_wouldent_fit_on_the_same_line
			);```
Is the proper way to do it (one more indentation for the lines inside the parenthesis) so i've fixed it. Will be deployed soon.
Also, comma on line start has been fixed, make sure you are using the latest norminette version @Tishj 
Thanks for the issue.

**Comment by N0ich** [comment]
Fixed with 3.0.5
